api/versioninfo.php

```diff
@@ -0,0 +1,117 @@
+<?php
+  /**
+    Finds the latest versions of downloads and returns a JSON blob with the data. The first *.download_info file in each folder will be checked.
```

Comment:
The errors are because the folders are not present in the repo. They are present on the live site... we could add the folders :)